src/App.tsx

```diff
@@ -5,7 +5,21 @@ import { FindMovie } from './components/FindMovie';
 import { Movie } from './types/Movie';
 
 export const App = () => {
-  const [movies] = useState<Movie[]>([]);
+  const [movies, setMovies] = useState<Movie[]>([]);
+  const [isMovieNew, setIsMovieNew] = useState(true);
+
+  const addMovie = (movie: Movie) => {
```

Comment:
keep best practice naming
```suggestion
  const handleAddMovie = (movie: Movie) => {
```

Reply:
check it throughout your code

Reply:
We don`t handle  something specific which occurs on and up to action, I pass it as props, I don`t call it in current component